polliwog/transform/rigid_transform.py

```diff
@@ -8,18 +11,18 @@ def find_rigid_transform(a, b, visualize=False):
     Based on Arun et al, "Least-squares fitting of two 3-D point sets," 1987.
```

Comment:
R going to be is a 3x3 rotation matrix and T is a 1x3 translation vector, is that right?

Also, am I right from this explanation that the verts are meant to be in correspondence? It would be good to clarify that. It makes me wonder if this ought to be moved to [entente](https://github.com/lace/entente).